### PR TITLE
Paololeg argentina

### PR DESCRIPTION
**Pull request by waynehiggins**
Im not sure what is happening...  I hope someone can read this.

### CODE REVIEW

**Comment by waynehiggins** [comment]
Dont have a clue what I am doing... sigh!

**Comment by dipstar** [comment]
Looks like the CI builds have failed. Wonder if there is a way to manually get them going again.

File: _pins/paololeg-argentina.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: paololeg
+latitude: -26.844606
+longitude :65.225078
```
Comment (hectorsector) [inline _pins/paololeg-argentina.json]: @waynehiggins check the spacing here! It should say: `longitude: 65.225078`

Comment (ZenMindedBiz) [reply _pins/paololeg-argentina.json]: @hectorsector Nice catch! 👍 
@waynehiggins Almost pinned to the map!  🎉

**Comment by ZenMindedBiz** [comment]
I wonder if you can use paololeg as your githubhandle for this .json file when it looks like your githubhandle is @waynehiggins; we'll see.

We're all learning here... very cool to try something new to master.  😄

**Pull request merged**